src/DashboardHub/Bundle/AppBundle/Resources/views/Github/events.html.twig

```diff
@@ -0,0 +1,22 @@
+<div class="panel panel-default">
+    <div class="panel-heading">Github Events</div>
+
+    <table class="table hideOverflow">
+        <thead>
+        <tr>
+            <th>Type</th>
```

Comment:
These may need to be reviewed as not overly descriptive.

Are fine for now though